examples/apps/echo/client/echo_client.py

```diff
@@ -237,10 +239,24 @@ def Main(args=None):
 		json.dumps(res, indent=4)
 	))
 	if "result" in res:
-		decrypted_res = utility.decrypted_response(
-			json.dumps(res), session_key, session_iv)
-		logger.info("\nDecrypted response:\n {}".format(decrypted_res))
+		sig_obj = signature.ClientSignature()
+		status = sig_obj.verify_signature(json.dumps(res), worker_obj)
+		try:
+			if status == SignatureStatus.PASSED:
+				logger.info("Signature verification Successful")
+				decrypted_res = utility.decrypted_response(
+					json.dumps(res), session_key, session_iv)
```

Comment:
for function parameter, use json dictionary instead of json string

Reply:
Changed